core/iwasm/aot/aot_loader.c

```diff
@@ -1711,7 +1741,6 @@ is_literal_relocation(const char *reloc_sec_name)
     return !strcmp(reloc_sec_name, ".rela.literal");
 }
 
-#if defined(BH_PLATFORM_WINDOWS)
 static bool
 str2uint32(const char *buf, uint32 *p_res)
```

Comment:
what is the difference between `str2uint32` to the `strtol` from libc ?

Reply:
@lum1n0us  Seems the result of str2uint32 is a `raw hex` rather than an integer

Reply:
Yes, it is raw int bits of f32 and f64.